resources/user-guides/launch-data-elements.md

```diff
@@ -0,0 +1,86 @@
+
+4. Select the Data Element Type. 
+
+   Add the required configuration parameters. Most data element types do not require any configuration.
+
+5. Click Save.
+
+For example, to create a data element which maps to the Experience Cloud ID, from the Create New Data Element page, select Extension *Mobile Core*, Data Element Type *Experience Cloud ID* and enter Name *ECID*. 
+
+![create ECID data element](../../.gitbook/assets/data-elements-create-data-element-ecid.png)
+
+## Data Element Usage
+
+You can use data elements when defining rules in the Experience Platform Launch interface. When a rule configuration allows the use of data elements, a cylinder icon ![cylinder icon](../../.gitbook/assets/data-elements-cylinder.png) is displayed next to the configuration text field. Clicking the icon gives a list of data elements defined in the mobile property.
+
+![data element selection](../../.gitbook/assets/data-elements-select-data-element.png)
+
+Here is an example which creates a rule to send a postback containing the Experience Cloud ID when the application launches.
+
+1. From a Property page, open the Rules tab, then click Create New Rule.
+
+   ![create rule](../../.gitbook/assets/data-elements-create-rule.png)
+
+2. Give a unique, descriptive name to the rule.
+
+3. Add an Event configuration. 
+
+   Select Extension *Mobile Core* and Event Type *Launched*.
+
+   ![event configuration](../../.gitbook/assets/data-elements-event-configuration.png)
+
+4. Add a Condition configuration. 
+
+   Select Extension *Mobile Core* and Condition Type *Data Element*. Select the cylinder icon ![cylinder icon](../../.gitbook/assets/data-elements-cylinder.png) next to the "Data element" text field, then select the ECID data element we created above. Select *exists* as the condition.
+
+   ![condition configuration](../../.gitbook/assets/data-elements-condition-configuration.png)
+
+5. Add an Action configuration. 
+
+   Select Extension *Mobile Core* and Action Type *Send Postback*. In the "URL" text box, add a sample URL, for example *https://my.company.com/launch?ecid=*. Click the cylinder icon ![cylinder icon](../../.gitbook/assets/data-elements-cylinder.png) and select the ECID data element which adds a token for the data element by the cursor in the text box. When the rule is triggered, the token is replaced by the unique ECID value for the current user.
+
+   ![action configuration](../../.gitbook/assets/data-elements-action-configuration.png)
+
+6. Click Save.
+
+Once the property is [published](https://aep-sdks.gitbook.io/docs/getting-started/create-a-mobile-property#publish-configuration), this new rule is made available for download by applications [configured for this property](https://aep-sdks.gitbook.io/docs/getting-started/initialize-the-sdk#configure-the-sdk-with-an-environment-id). When the application launches, this rule is triggered and, if the ECID exists in the SDK, a postback is sent to the URL with the included ECID value.
```

Comment:
Please reword as follows:
"After the property is published, this new rule is made available for downloading by the applications that are configured for this property. For more information, see [Publish the configuration](https://aep-sdks.gitbook.io/docs/getting-started/create-a-mobile-property#publish-configuration) and [Configure the SDK with an Environment ID](https://aep-sdks.gitbook.io/docs/getting-started/initialize-the-sdk#configure-the-sdk-with-an-environment-id). When the application launches, this rule is triggered and, if the ECID exists in the SDK, a postback is sent to the URL with the ECID value."